docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -99,7 +99,7 @@ To install additional python dependencies that may be needed, with the example o
 
 .. code-block:: console
 
-    $ pipx inject ansible argcomplete
+    $ pipx inject --include-binaries ansible argcomplete
```

Comment:
`--include-binaries` doesn't exist. I think you want `--include-apps`.

Reply:
Oh I see. Thank you for catching that. Looks like this is dependent on the version of `pipx` unfortunately, and I was working off Ubuntu 20.04 which ships an older version of pipx. `--include-binaries` was replaced in 0.14.0.0 by ``--include-apps`.

I will update.

Reply:
You switched the wrong line. `pipx install --include-deps ansible` was correct. `pipx inject --include-binaries ansible argcomplete` was not.

Reply:
Oh dear, that is quite embarrassing!

 I'm sorry, I think I need to get some more tea this morning.